test/libsinsp_e2e/resources/docker/large_container_dockerfiles/Dockerfile.long_labels

```diff
@@ -0,0 +1,14 @@
+FROM busybox
```

Comment:
Please address this comment https://github.com/falcosecurity/libs/pull/1720/files#r1509470217

Reply:
First assign to an ENV and then the ENV to the label.  Is this also not possible?

Reply:
That doesn't work either. I think that docker treats them as simple strings and it makes sense: in case bash or sh are not present, it will not know how to expand them.

Reply:
https://docs.docker.com/reference/dockerfile/#label

```
Be sure to use double quotes and not single quotes. Particularly when you are using string interpolation (e.g. LABEL example="foo-$ENV_VAR"), single quotes will take the string as is without unpacking the variable's value.
```

Or pass to runtime, then we surely have bash available?

Reply:
I'm afraid that isn't possible either :/ https://github.com/moby/moby/issues/29110

Reply:
What about https://superuser.com/questions/1559432/adding-automatic-labels-as-part-of-docker-build#:~:text=In%20the%20build%20command%2C%20you,the%20Dockerfile%20for%20that%20label.

```
docker build --label "build-agent=${NODE_NAME}" -t $image_name .
```

or when launching the container, but above during building the container should work?

On that note we have like 8 Dockerfiles -- all busybox even; we could just have one that contains `FROM busybox` and few other things we always use or need and then have one script that builds the various container variants we need for testing. `docker build --build-arg` might be of interest too. 

Got to go soon, I might have time on Sunday to try hands-on myself, but not today or tomorrow.

Reply:
Ok just had a moment to try:

Here is my recommendation: Have just one Dockerfile with a default HEALTHCHECK we use the most.

Choose your preferred approach to ad-hoc create the labels that are assigned to an ENV. For illustration I used just bash

```bash
REPEATED_CHARACTER=a
REPEAT_COUNT=100
LONG_STRING=$(printf "%0.s$REPEATED_CHARACTER" $(seq 1 $REPEAT_COUNT))

docker build --label "url2=${LONG_STRING}" -f Dockerfile -t test:latest .

docker image inspect --format='{{json .Config.Labels}}' test:latest
{"url2":"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"}
```

According to https://stackoverflow.com/questions/53772248/how-to-modify-docker-health-check-without-rebuilding-image

```
--health-cmd            Command to run to check health
--health-interval       Time between running the check
--health-retries        Consecutive failures needed to report unhealthy
--health-timeout        Maximum time to allow one check to run
--health-start-period   Start period for the container to initialize before starting health-retries countdown
--no-healthcheck        Disable any container-specified HEALTHCHECK
```

... the healthcheck should be adjustable when running the container:

```bash
docker run --health-cmd="/bin/ut-health-check" --health-interval=0.6s test:latest
```
Effectively we could also really just build one container and also pass the `--label` when launching the container. That way it would be just one approach to invoke the correct test respectively.

I believe this would be easier to maintain long term as we will likely expand the container e2e tests to crictl and more ...

Reply:
I address this point adding a small helper class and removed all dockefiles except the base one and 2 more needed to test the presence of health checks: I didn't find a way to remove the health checks or setting them to `NONE` from command line. I hope that it is enough :)

Reply:
oh that's disappointing that the healthcheck args passed when running the container are not working ... yes down to 3 from 8 Dockerfiles is an improvements. We can always improve it more in the future.